napari/_qt/qt_about.py

```diff
@@ -0,0 +1,48 @@
+import skimage
+import vispy
+import scipy
+import numpy
+
+import qtpy
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QWidget, QVBoxLayout, QLabel, QDialog, QFrame
+
+import napari
+
+
+class AboutPage(QWidget):
+    def __init__(self, parent):
+        super(AboutPage, self).__init__(parent)
+
+        self.layout = QVBoxLayout()
+
+        # Description
+        self.layout.addWidget(
+            QLabel("<b>napari</b>: a fast n-dimensional image viewer")
+        )
+
+        # Horizontal Line Break
+        self.hline_break1 = QFrame()
+        self.hline_break1.setFrameShape(QFrame.HLine)
+        self.hline_break1.setFrameShadow(QFrame.Sunken)
+        self.layout.addWidget(self.hline_break1)
+
+        # Versions
+        self.layout.addWidget(QLabel("napari, " + napari.__version__))
```

Comment:
Is there any way to make this more selectable text then just the qlabels. i.e. when it pops open I'd like to be able to drag and select all the text so i could easily paste it into a github issue, now I need to take a screenshot

Reply:
this is easy, im changing as you requested